e2e-tests/portal/cypress/support/portal/createBssDeal.js

```diff
@@ -0,0 +1,130 @@
+// This file is used to create a BSS application that is ready for check
+const { submissionDetails } = require('../../fixtures/deal');
+const {
+  dashboard,
+  selectScheme,
+  beforeYouStart,
+  bankDetails,
+  contract,
+  eligibilityCriteria,
+  eligibilityPreview,
+  bondDetails,
+  bondFinancialDetails,
+  bondFeeDetails,
+  contractReadyForReview,
+  contractAboutSupplier,
+  contractAboutBuyer,
+  contractAboutFinancial,
+} = require('../../e2e/pages');
+
+const login = require('./login');
+const MOCK_USERS = require('../../../../e2e-fixtures/index');
+
+const { BANK1_MAKER1 } = MOCK_USERS;
+
+/**
+ * Create a BSS/EWCS deal via the UI.
+ * @param {Boolean} readyForCheck: Conditionally complete all "maker" required forms
+ * @param {Boolean} unissuedFacilities: TBD
+ */
+
+const createBssDeal = ({ readyForCheck = false, unissuedFacilities = false }) => {
+  // Log in as BANK1_MAKER1
+  login(BANK1_MAKER1);
```

Comment:
Does this create a specific BSS/EWCS deal can this please be reflected in the docstring.

Reply:
Sorry, this was added by accident from stashing/branch switching - have removed.